README.md

```diff
@@ -64,17 +64,28 @@ OpenPAI is a most complete solution for deep learning, support virtual cluster,
 
 OpenPAI manages computing resources and is optimized for deep learning. Through docker technology, the computing hardware are decoupled with software, so that it's easy to run distributed jobs, switch with different deep learning frameworks, or run other kinds of jobs on consistent environments.
 
-As OpenPAI is a platform, there might be two different roles who are involved. One is the cluster administrator, and the other is the cluster user. We provide end-to-end manuals for different roles.
+As OpenPAI is a platform, there are typically two different roles
 
-### Manual for cluster administrators
+- **Cluster users** are the consumer of the cluster's computing resources. According to the deployment scenarios, cluster users could be researchers of Machine Learning and Deep Learning, data scientists, lab teachers and students and so on.  
```

Comment:
consumer -> consumers; 

lab teachers and students and so on -> lab teachers, students, and so on.